src/data/meetups.ts

```diff
@@ -1,68 +1,158 @@
   {
     url: "https://www.meetup.com/techtank-to/",
     platform: "meetup",
-    type: "meetups"
+    type: "meetups",
   },
   {
     url: "https://www.meetup.com/torontojs/",
     platform: "meetup",
-    type: "meetups"
+    type: "meetups",
   },
   {
     url: "https://www.meetup.com/go-toronto/",
     platform: "meetup",
-    type: "meetups"
+    type: "meetups",
   },
   {
     url: "https://www.meetup.com/toronto-java-users-group/",
     platform: "meetup",
-    type: "meetups"
+    type: "meetups",
   },
   {
     url: "https://www.meetup.com/k8s-ca/",
     platform: "meetup",
-    type: "meetups"
+    type: "meetups",
   },
   {
     url: "https://www.meetup.com/drupalto/",
     platform: "meetup",
-    type: "meetups"
+    type: "meetups",
   },
   {
     url: "https://www.meetup.com/toronto-aws-users-united/",
     platform: "meetup",
-    type: "meetups"
+    type: "meetups",
   },
   {
     url: "https://www.meetup.com/elastic-toronto-user-group/",
     platform: "meetup",
-    type: "meetups"
+    type: "meetups",
   },
   {
     url: "https://www.meetup.com/toronto-tech-stack-exchange/",
     platform: "meetup",
-    type: "meetups"
+    type: "meetups",
   },
   {
     url: "https://www.meetup.com/machine-learning-to-meetup/",
     platform: "meetup",
-    type: "meetups"
+    type: "meetups",
   },
   {
     url: "https://toronto-ruby.com/",
     platform: "other",
-    type: "meetups"
+    type: "meetups",
   },
   {
     url: "https://builder-sundays.myshopify.com/",
     platform: "other",
-    type: "others"
-  }
+    type: "others",
+  },
+  {
+    url: "https://www.meetup.com/brainstation-toronto-tech-skills-and-careers/",
+    platform: "meetup",
+    type: "meetups",
+  },
+  {
+    url: "https://www.meetup.com/metro-toronto-azure-community/",
+    platform: "meetup",
+    type: "meetups",
+  },
+  {
+    url: "https://www.meetup.com/pragmatic-tech/",
+    platform: "meetup",
+    type: "meetups",
+  },
+  {
+    url: "https://www.meetup.com/laravel-toronto/",
+    platform: "meetup",
+    type: "meetups",
+  },
+  {
+    url: "https://www.meetup.com/vue-toronto/",
+    platform: "meetup",
+    type: "meetups",
+  },
+  {
+    url: "https://www.meetup.com/toronto-modern-data/",
+    platform: "meetup",
+    type: "meetups",
+  },
+  {
+    url: "https://www.meetup.com/cloud-architecture-meetup-group/",
+    platform: "meetup",
+    type: "meetups",
+  },
+  {
+    url: "https://www.meetup.com/aittg-toronto/",
+    platform: "meetup",
+    type: "meetups",
+  },
+  {
+    url: "https://www.meetup.com/digitalnatives/",
+    platform: "meetup",
+    type: "meetups",
+  },
+  {
+    url: "https://www.techinto.ca/",
+    platform: "other",
+    type: "meetups",
+  },
+  {
+    url: "https://www.meetup.com/mindstone-toronto-ai-meetup/", // "https://community.mindstone.com/events"
+    platform: "meetup",
+    type: "meetups",
+  },
+  {
+    url: "https://www.genaicollective.ai/chapters/toronto",
```

Comment:
Currently, we use these scripts to scrape data, and they need to be customized for each website if it's not a Meetup page. I'm working on a better long-term approach, but for now, the scripts have to be modified manually to support sources other than Meetup or the already added pages.
refs. https://github.com/Sean0628/gta-dev-web/tree/master/scripts

So, do you want to add support for these new pages now, or would it be okay to just include Meetup links for the time being and add support for the others later?

Reply:
Either works for me.